schema/generator

```diff
@@ -34,7 +34,7 @@ fi
 $GENDOC --f $IN --o $DOC
 
 # See schema/generator_import.go for instructions on updating the dependency
```

Comment:
please add to the comment how this fixes it, for the benefit of future generations